docs/monitors/elasticsearch-query.md

```diff
@@ -0,0 +1,285 @@
+<!--- GENERATED BY gomplate from scripts/docs/templates/monitor-page.md.tmpl --->
+
+# elasticsearch-query
+
+Monitor Type: `elasticsearch-query` ([Source](https://github.com/signalfx/signalfx-agent/tree/master/pkg/monitors/elasticsearch/query))
+
+**Accepts Endpoints**: **Yes**
+
+**Multiple Instances Allowed**: Yes
+
+## Overview
+
+This monitor metricizes aggregated responses from Elasticsearch. The monitor
+constructs SignalFx datapoints based on Elasticsearch aggregation types and
+also aggregation names.
+
+An simple configuration looks like the following:
+
+```yaml
+monitors:
+- type: elasticsearch-query
+ host: localhost
+ port: 9200
+ index: <name_of_index>
+ elasticsearchRequest: |
```

Comment:
You would probably want to set `intervalSeconds: 300` on this example, otherwise the CPU utilization will be overlapping.  Or change 5m to 20s.